### DISCUSSION TITLE
VDH greyed out on some youtube pages

### DISCUSSION THREAD

**Discussion by Dave-Rado** [aclap-dev/video-downloadhelper#918] [Help and Q&A]
### Your question

I tried to download the video at https://www.youtube.com/watch?app=desktop&v=juQL-0db_wc&ab_channel=CountAmadeusProductions but the VDH icon is greyed out. Any idea why?

Dave

PS - when I went into settings to copy the version details, it said it was licensed to an email address that is 6 years out of date! How can I update the licence to my current email address?

### Addon & CoApp version and details

Version 9.0.2.12 (stable), target mozilla, locale en-GB.

Companion App installed (v2.0.19)

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3737, Firefox 127.0.1 64-bit, licensed VDH 9.0.2.12a1 beta, CoApp 2.0.19.  You should post all of that information.  Or you should just go into the `More settings` of VDH, click the button that says `Copy information to clipboard`, & paste all of that into your problem description.  One or the other.  You should always include all of those bits of information.

I visited your YouTube page & got this:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/4bbb959f-c5b2-48fc-9aa1-a35ccdac8d3c)

Looks OK to me.  Normally, I'd be tempted to get an opera.  But the resolution on this one is . . . well, I can't candy-coat it . . . putrid.  I already have 3 performances of Die Zauberflöte from the Met in HD in my archive so this one gets a pass from me.  Meaning I'm not going to download a 2 & 1/2 hour video as a test.  But it looks like VDH would have no problem with this one.

The major difference between your setup & mine is that you are not running the beta while I am.  It is a good idea to always use the beta.  The VDH beta is not, as is true with so much other software available elsewhere, an experimental bug-filled disaster waiting to happen.  The VDH betas have nearly always been totally reliable.  They contain bug fixes that people really need to have before they report problems.  It seems the beta must contain a bug fix somewhere that is missing from the stable release.  So if you want this opera, you'll install the beta.  You'll find the beta in the Wiki, whose link appears at the top of every page of this forum.

**Reply by GrampaWildWilly** [reply]
For your E-mail licensing thing, you should post something in the section of this forum devoted to that topic.  You should look in the Wiki for information on that.  You'll probably find that, instead of posting something here, there is an E-mail address you should send an E-mail to detailing your situation.
